oracle/fuel-toolchain.toml

```diff
@@ -0,0 +1,6 @@
+[toolchain]
+channel = "beta-1"
```

Comment:
```suggestion
channel = "latest-2023-01-18"
```

I don't know about that one chief

Reply:
Ughh I was testing locally, thanks for catching this.